scripts/deploy-latest.sh

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env bash
+
+# Environment variables:
+# 1. METAKGP_WIKI_PATH => Path to the cloned metakgp/metakgp-wiki repository
+# 2. SLACK_NOTIFICATIONS_URL => Webhook URL to which notifications will be sent as a Post request
+
+# set -x
```

Comment:
We want `set -e` I think?

Reply:
Actually, I am using the exit code from `git --no-pager diff --exit-code $deploy_branch > /dev/null` to check whether there is any diff between the current version and the future version.

```
	git --no-pager diff --exit-code $deploy_branch > /dev/null

	if [[ "$?" == "0" ]];
	then
		echo "$base_branch and $deploy_branch are the same. Continue with deployment? (y/N)"
```

I am guessing that we can also look at the actual `git diff` output, but I prefer the exit code as it removes the need for `$(git diff ... | wc -l) -gt 0` type if-else blocks. Would you prefer that? (I don't have strong opinions, so I don't mind changing this to `set -e` and ensuring that all commands will return with 0 exit code. :ok_man: )

[1]: http://www.linuxcommand.org/lc3_man_pages/seth.html